src/Spinners/LoadSpinner.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { BaseTheme } from '../Theme';
+
+
+export interface LoadSpinnerProps {
+  /** Allows you to pass in style properties for the element */
+  theme?: object;
+  /** Specifies the text under the spinner */
+  children: ReactElement;
+}
+
+const StyledLoadSpinner = styled.div`
+  background: transparent;
+  color: ${({ theme }): string => (theme.color.background.dark)};
```

Comment:
We actually have a separate set of color values for text, and I think our `info` color is probably close to what the spinner in the older course-planner is, so this should be:

```suggestion
  color: ${({ theme }): string => (theme.color.text.info)};
```